mvc/overview.rst

```diff
@@ -1,8 +1,68 @@
 
-.. _issue: https://github.com/aspnet/Docs/issues/113
+- :doc:`Views </views/index>` are responsible for displaying the application's user interface (UI). Typically, this UI is created from the model data. An example would be an edit view of a Products table that displays text boxes, drop-down lists, and check boxes based on the current state of the Product object being represented in the view.
+
+- :doc:`Controllers </controllers/index>` process incoming requests, work with the model, and ultimately select a view that displays the user interface. In an MVC application, the view only displays information; the controller handles and responds to user input and interaction. For example, the controller handles query-string values, and passes these values to the model, which in turn might use these values to query the database.
+
+The MVC pattern helps you create applications that separate the different aspects of the application (input logic, business logic, and UI logic), while providing a loose coupling between these elements. The pattern specifies where each kind of logic should be located in the application:
+
+- UI logic belongs in the view
+- Input logic belongs in the controller
+- Business logic belongs in the model
+
+This delineation of responsibilities – referred to as the `separation of concerns (SoC) <https://en.wikipedia.org/wiki/Separation_of_concerns>`_ - helps you scale the application in terms of complexity because it’s much easier to code, debug, and test something (model, view, or controller) that has a single job rather than something that performs multiple tasks.
+
+Features of the ASP.NET MVC framework
+-------------------------------------
+
+The ASP.NET MVC framework provides the following features:
+
+- Separation of application tasks (input logic, business logic, and UI logic), testability, and test-driven development (TDD). All core contracts in the MVC framework are interface-based and can be tested by using mock objects, which are simulated objects that imitate the behavior of actual objects in the application. You can unit-test the application without having to run the controllers in an ASP.NET process, which makes unit testing fast and flexible. You can use any unit-testing framework that is compatible with the .NET Framework.
+- An extensible and pluggable framework. The components of the ASP.NET MVC framework are designed so that they can be easily replaced or customized. You can plug in your own view engine, URL routing policy, action-method parameter serialization, and other components. The ASP.NET MVC framework also supports the use of Dependency Injection (DI) and Inversion of Control (IoC) container models. DI enables you to inject objects into a class, instead of relying on the class to create the object itself. IOC specifies that if an object requires another object, the first objects should get the second object from an outside source such as a configuration file. This makes testing easier.
```

Comment:
This isn't really an accurate definition of IoC (see the [Wikipedia definition](https://github.com/aspnet/Docs/pull/440) which is pretty good) - also it's confusing IoC with IoC _containers_, Maybe just say DI is built in and you can use your choice of IoC containers?